TrelloAutotest/Pages/TrelloBoardPage.cs

```diff
@@ -11,33 +11,30 @@ public class TrelloBoardPage : BasePage
     {
         public void OpenBoardMenu() => driver.FindElements(By.TagName("a")).First(x => x.Text.Contains(MessageText.ShowMenu)).Click();
 
-       
-
-        public TrelloBoardPage IsUrlCorrect(string BoardName)
+        public TrelloBoardPage ValidateUrl(string BoardName)
         {
             Wait.Until(d => driver.FindElements(BoardSelectors.BoardViewButton).Any());
 
-            var SplitName = BoardName.Contains(' ') ? BoardName.ToLower().Replace(' ', '-').TrimEnd('-') : BoardName.ToLower();
-            var url = TrimUrl(driver.Url);
+            var splitedName = (BoardName.Contains(' ') ? BoardName.Replace(' ', '-').TrimEnd('-') : BoardName).ToLower();
+            var url = GetBoardNameUrl(driver.Url);
 
-            Assert.IsTrue(url == (SplitName));
+            Assert.IsTrue(url == splitedName);
             return this;
         }
 
-        public string TrimUrl(string Url) => Url.Split('/').Last();
+        private string GetBoardNameUrl(string Url) => Url.Split('/').Last();
 
-        public TrelloBoardPage IsBoardCreated(bool assertFlag)
+        public TrelloBoardPage ValidateBoardCreating(bool expectedResult)
```

Comment:
1. There is a difference between validation and verification - you're doing verification here
2. Method name and method body don't match.
3. For having test case step more clear, I would split this method into 2 separate - for IsTrue and IsFalse

Reply:
Apply this comment to all of the changes